src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/AdditionalFiles.BrowseObject.xaml

```diff
@@ -140,7 +140,7 @@
     </EnumProperty.DataSource>
     <EnumValue Name="Designer" />
     <EnumValue Name="Component" />
-    <EnumValue Name="Control" />
+    <EnumValue Name="UserControl" />
     <EnumValue Name="Form" />
```

Comment:
This is also wrong, this doesn't exist.

Reply:
`Form`? Okay, I shall remove.

Reply:
Where is the official list of subtypes? `Form` is used in [our own code](https://github.com/dotnet/project-system/blob/afa02d302e08df1ffe8689dbcd6b9d65c6f59c2f/src/Microsoft.VisualStudio.AppDesigner/Microsoft.VisualStudio.AppDesigner.vbproj#L50) in a couple of spots, and is part of legacy: http://index/?query=%60VSSUBTYPE_FORM&rightProject=VS(Repo)&file=src%5Cvsproject%5Cinc%5Csubtypes.h&line=25

Reply:
Yeah my bad, I even handled this in https://github.com/dotnet/project-system/pull/4913. I was originally under the impression that Form/Designer were the same, but they aren't: http://index/?query=Form&rightProject=System.Windows.Forms&file=System%5CWinForms%5CForm.cs&line=49. Let me find which control uses "Designer".